product_docs/docs/pgd/3.7/bdr/release-notes.mdx

```diff
@@ -5,6 +5,53 @@ originalFilePath: release-notes.md
 
 ---
 
+## BDR 3.7.22 (2023 Aug 29)
+
+This is a maintenance release for BDR 3.7 that includes minor
+improvements as well as fixes for issues identified in previous
+versions.
+
+Also check the release notes for [pglogical 3.7.22](/pgd/3.7/pglogical/release-notes/#pglogical-3722) for resolved issues
+that affect BDR as well.
+
+
+### Resolved issues
+
+- Changed `bdr.autopartition_drop_partition()` signature to use text.
+
+- Autopartition: Drop partition iff it exists
```

Comment:
```suggestion
- Autopartition: Drop partition if it exists
```

Reply:
"If and only if"

Reply:
I suspect many customers reading the release notes may not be familiar enough with logic and mathematics to understand this term. It would be better to spell it out (Kelly also pointed this out).